nautilus_trader/adapters/interactive_brokers/historical/client.py

```diff
@@ -60,6 +60,7 @@ def __init__(
         client_id: int = 1,
         market_data_type: MarketDataTypeEnum = MarketDataTypeEnum.REALTIME,
         log_level: str = "INFO",
+        cache: Cache | None = None,
```

Comment:
Would it be better to pass a `cache_config` here instead?
https://github.com/nautechsystems/nautilus_trader/blob/develop/nautilus_trader/cache/config.py#L23.

Similar to https://github.com/nautechsystems/nautilus_trader/blob/develop/nautilus_trader/system/config.py#L109